docs/developer/architecture/identity-trust-protocol/identity-and-trust-modules.md

```diff
@@ -0,0 +1,93 @@
+# Modules and services of the IdentityHub
+
+![module-overview](./identity.hub.modules.png)
+
+## VC Module
+
+- contains the `PresentationCreationService` and a `CredentialStateMachine`.<br/>
```

Comment:
I can't remember if we discussed this, and we can change the name later, but I would not call this `CredentialStateMachine` in the code, .cf `TransferProcess`, `ContractNegotiation` have state machines but are entities.

Reply:
renamed it to `VerifiableCredentialManager` so that it corresponds with the architecture doc.